Basic-Video-Chat/app/build.gradle

```diff
@@ -1,34 +1,40 @@
-apply plugin: 'com.android.application'
-apply from: '../../build.properties'
+plugins {
+    id 'com.android.application'
+}
+
+apply {
+    from '../../build.properties'
+}
```

Comment:
just curious but why are you changing these lines to the `{}` format?

Reply:
I wanted to match the new Gradle format for applying plugins:

```
plugins {
    pluginA
}
```

but now when I am thinking about this - prev approach was better. 
I will revert this in following PRs

Reply:
Just to match the new Gradle format for applying plugins (that nowadays is used by AS when creating a new project):

![image](https://user-images.githubusercontent.com/530122/104597565-c5862f80-5675-11eb-9f06-b7030ca0103a.png)

Both will work